From: Hannes Reinecke <hare@suse.de>
Date: Wed, 17 Jul 2019 13:24:09 +0200
Subject: [PATCH] scsi/fc: kABI fixes for new ELS_FPIN definition
Patch-Mainline: never, kABI fix for SLE15-SP1
References: bsc#1136217,jsc#SLE-4722

Signed-off-by: Hannes Reinecke <hare@suse.com>
[dwagner: Added ELS_RDF definitions]
Signed-off-by: Daniel Wagner <dwagner@suse.de>
---
--- a/include/uapi/scsi/fc/fc_els.h
+++ b/include/uapi/scsi/fc/fc_els.h
@@ -53,7 +53,9 @@ enum fc_els_cmd {
 	ELS_REC =	0x13,	/* read exchange concise */
 	ELS_SRR =	0x14,	/* sequence retransmission request */
 	ELS_FPIN =	0x16,	/* Fabric Performance Impact Notification */
+#ifndef __GENKSYMS__
 	ELS_RDF =	0x19,	/* Register Diagnostic Functions */
+#endif
 	ELS_PRLI =	0x20,	/* process login */
 	ELS_PRLO =	0x21,	/* process logout */
 	ELS_SCN =	0x22,	/* state change notification */
